sourced/ml/__main__.py

```diff
@@ -35,6 +35,45 @@ def one_arg_parser(*args, **kwargs) -> argparse.ArgumentParser:
     return arg_parser
 
 
+def add_spark_args(my_parser):
+    my_parser.add_argument(
+        "-s", "--spark", default=SparkDefault.MASTER_ADDRESS,
+        help="Spark's master address.")
+    my_parser.add_argument(
+        "--config", nargs="+", default=SparkDefault.CONFIG,
+        help="Spark configuration (key=value).")
+    my_parser.add_argument(
+        "-m", "--memory",
+        help="Handy memory config for spark. -m 4,10,2 is equivalent to "
```

Comment:
I think it's better to forward `4G` instead of just `4`.
Just in case if someone in future will need less than 1G. And it's only 1 letter more.

Reply:
it's not clear how it will interfere with `--config` arguments above.

Reply:
I think Konst copied this from existing code. Let's create an issue and fix it later.

Reply:
it was @vmarkovtsev's request, I am fine if vadim agree to add `G` letter.

Reply:
No objections.